go.mod

```diff
@@ -31,6 +31,7 @@ require github.com/tredoe/osutil v1.0.6
 
 require (
 	github.com/chigopher/pathlib v1.0.0 // indirect
+	github.com/d-tux/go-fstab v0.0.0-20141204152952-eb4090f26517 // indirect
```

Comment:
Just golang  curiosity, why is this `indirect`?

Reply:
An indirect dependency is a dependency added but not used, I added the dependency via go get, and I was not using before I wrote the code.

After you wrote the code the dependency is used and it's not indirect anymore, but go build and go run does not update the go mod file.

You should call directly go mod tidy to fix this type of inconsistency in the go mod, I've done that, thanks for spotting! :D